introduction.md

```diff
@@ -1,4 +1,5 @@
 Introduction
 ==========
 * my name is Emmanuel Lopez, I am learning github in master github. "codeschool"
-* my favorite codeschool path is JavaScript, HTML & CSS, iOS
\ No newline at end of file
+* my favorite codeschool path is JavaScript, HTML & CSS, iOS
+* this is deadly_skills branch change.
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?